### PR TITLE
flameshot: Include patch to fix wayland/sway scale-issue

### PR DESCRIPTION
**Pull request by SebTM**
###### Description of changes

Fix scaling issue should be included in one of the next releases

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by oxalica — APPROVED** [review]
> Fix scaling issue should be included in one of the next releases

Why? The [flameshot PR](https://github.com/flameshot-org/flameshot/pull/3166) seems not merged yet.

Anyway, the patch itself works for me on sway with fractional scale.

---
Result of `nixpkgs-review pr 237987` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>flameshot</li>
  </ul>
</details>

File: pkgs/tools/misc/flameshot/default.nix [hunk 1]
```diff
@@ -19,6 +20,14 @@ mkDerivation rec {
     sha256 = "sha256-omyMN8d+g1uYsEw41KmpJCwOmVWLokEfbW19vIvG79w=";
   };
 
+  patches = [
+    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/flameshot/default.nix:24]:
```suggestion
    # https://github.com/flameshot-org/flameshot/pull/3166
    (fetchpatch {
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
otherwise a force push to the PR breaks the hash

**Comment by Janik-Haag** [comment]
@SebTM can you apply the changes Sandro suggested so we can get this merged? Should be a 5min task.

**Comment by SebTM** [comment]
> @SebTM can you apply the changes Sandro suggested so we can get this merged? Should be a 5min task.

Done, was on vacation 🌴✌🏻

**Review by Janik-Haag — APPROVED** [review]

**Pull request merged**